ferrischat_webserver/src/guilds/edit_guild.rs

```diff
@@ -0,0 +1,44 @@
+use actix_web::web::Json;
+
+use actix_web::{HttpRequest, HttpResponse, Responder};
+use ferrischat_common::request_json::GuildUpdateJson;
+use ferrischat_common::types::{Guild, InternalServerErrorJson, NotFoundJson};
+
+pub async fn edit_guild(
+    req: HttpRequest,
+    guild_info: Json<GuildUpdateJson>,
+    _: crate::Authorization,
+) -> impl Responder {
+    let guild_id = get_item_id!(req, "guild_id");
+    let bigint_guild_id = u128_to_bigdecimal!(guild_id);
+
+    let GuildUpdateJson { name } = guild_info.0;
+
+    let db = get_db_or_fail!();
+
+    let resp = sqlx::query!(
+        "UPDATE guilds SET name = $2 WHERE id = $1 RETURNING *",
+        bigint_guild_id,
+        name
```

Comment:
Same as above, please swap `$1` and `$2` in both the query and macro.

Reply:
fine